### ISSUE TITLE
Difference between anvil and smallvil

### ISSUE THREAD

**Issue by shoaibmerchant** [Smithay/smithay#852]
This maybe a redundant question, but I couldn't find any documentation on what are the differences between the anvil and smallvil reference compositors. I could see that anvil has xwayland support, but smallvil doesn't.

 If anybody can help elaborate this, I would be grateful. Thank you for all the efforts!

**Comment by shoaibmerchant** [comment]
Just to add here, I understand smallvil is a limited compositor, but it would be nice to see a feature/support breakdown somewhere in the readme.

**Comment by PolyMeilex** [comment]
Smallvil is meant to be as simple as possible, and implement only core protocol and xdg-shell, the idea is that it should be extensively commented, and simple to follow, the only reason for its existence is to be an example.

Anvil in comparison is a testing ground for most features available in Smithay, a lot of optional protocols, a lot of features, and sadly as a result a lot of code, that can be overwhelming to newcomers.

So tldr could be: Smallvil exists to lure newcomers in :smile: instead of scaring them off like Anvil does.

**Comment by shoaibmerchant** [comment]
Great that makes sense, thanks!
